### PR TITLE
pythonPackages.fuzzywuzzy: init at 0.17.0

### PR DESCRIPTION
**Pull request by rvolosatovs**
###### Things done

- [ ] Tested using sandboxing ([nix.useSandbox](http://nixos.org/nixos/manual/options.html#opt-nix.useSandbox) on NixOS, or option `sandbox` in [`nix.conf`](http://nixos.org/nix/manual/#sec-conf-file) on non-NixOS)
- Built on platform(s)
   - [x] NixOS
   - [ ] macOS
   - [ ] other Linux distributions
- [ ] Tested via one or more NixOS test(s) if existing and applicable for the change (look inside [nixos/tests](https://github.com/NixOS/nixpkgs/blob/master/nixos/tests))
- [ ] Tested compilation of all pkgs that depend on this change using `nix-shell -p nox --run "nox-review wip"`
- [x] Tested execution of all binary files (usually in `./result/bin/`)
- [ ] Determined the impact on package closure size (by running `nix path-info -S` before and after)
- [ ] Assured whether relevant documentation is up to date
- [x] Fits [CONTRIBUTING.md](https://github.com/NixOS/nixpkgs/blob/master/.github/CONTRIBUTING.md).

---

### CODE REVIEW

**Comment by dotlambda** [comment]
@GrahamcOfBorg build python2.pkgs.fuzzywuzzy python3.pkgs.fuzzywuzzy

File: pkgs/development/python-modules/fuzzywuzzy/default.nix [hunk 1]
```diff
@@ -0,0 +1,22 @@
...
+  };
+
+  propagatedBuildInputs = [
+    pycodestyle
+  ];
+
+  meta = with lib; {
+    description = "Fuzzy String Matching in Python";
+    homepage = https://github.com/seatgeek/fuzzywuzzy;
+    license = licenses.gpl20;
```
Comment (dotlambda) [inline pkgs/development/python-modules/fuzzywuzzy/default.nix]:
```suggestion
    license = licenses.gpl2;
```

**Review by dotlambda — CHANGES_REQUESTED** [review]

File: pkgs/development/python-modules/fuzzywuzzy/default.nix [hunk 2]
```diff
@@ -0,0 +1,22 @@
+{ lib , buildPythonPackage , fetchPypi, pycodestyle }:
```
Comment (dotlambda) [inline pkgs/development/python-modules/fuzzywuzzy/default.nix]:
```suggestion
{ lib, buildPythonPackage, fetchPypi, pycodestyle }:
```

**Review by dotlambda — CHANGES_REQUESTED** [review]

**Comment by worldofpeace** [comment]
This is already in https://github.com/NixOS/nixpkgs/pull/53261

**Comment by worldofpeace** [comment]
@rvolosatovs Sorry that you did this without knowing there was an open pr. We get a lot of prs for python that usually bring in a lot of dependents.

**Comment by rvolosatovs** [comment]
Happens, FWIW this was a "dependent" itself, I was working on adding a bigger package(which did not succeed at the end), so I extracted things that did work and opened a bunch of PRs today.

**Pull request closed without merging**